scripts/standby_node_health_check/standby_node_health_check.py

```diff
@@ -0,0 +1,401 @@
+#
+# Copyright © 2013-2024 NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+#
+# This software product is a proprietary product of Nvidia Corporation and its affiliates
+# (the "Company") and all right, title, and interest in and to the software
+# product, including all associated intellectual property rights, are and
+# shall remain exclusively with the Company.
+#
+# This software product is governed by the End User License Agreement
+# provided with the software product.
+#
+
+import re
+import subprocess
+import argparse
+import sys
+
+
+UFM_HA_CLUSTER_COMMAND = "ufm_ha_cluster {}"
```

Comment:
maybe class members is better than global variables.

Reply:
Done